src/components/ui/events/MainTable.js

```diff
@@ -32,6 +32,14 @@ mainTable.getEventInfo = function(eventRowNum){
     });
 };
 
+mainTable.checkAllMessagesToContain = function(text) {
+    return element.all(by.css('p.task')).getText().then(function(contents){
+        contents.forEach(function(message){
+            expect(message).toMatch(text);
```

Comment:
The except should not be in the service - which is really specific to a test.
A service should only return information/do action.